pkg/dogstatsd/enrich.go

```diff
@@ -69,9 +89,9 @@ func enrichMetricType(dogstatsdMetricType metricType) metrics.MetricType {
 	return metrics.GaugeType
 }
 
-func isBlacklisted(metricName, namespace string, namespaceBlacklist []string) bool {
+func isExcluded(metricName, namespace string, namespaceExcluded []string) bool {
 	if namespace != "" {
-		for _, prefix := range namespaceBlacklist {
+		for _, prefix := range namespaceExcluded {
 			if strings.HasPrefix(metricName, prefix) {
 				return true
 			}
```

Comment:
Why the change in naming ?

Reply:
Will make the change in a separate PR 👍

Reply:
https://github.com/DataDog/datadog-agent/pull/7559